examples/websocket_tls/sticky/server/server.go

```diff
@@ -27,9 +28,25 @@ func onWebsocket(w http.ResponseWriter, r *http.Request) {
 		panic(err)
 	}
 	wsConn := conn.(*websocket.Conn)
-	wsConn.OnMessage(func(c *websocket.Conn, messageType websocket.MessageType, data []byte) {
-		// echo
-		c.WriteMessage(messageType, data)
+	mtx := sync.Mutex{}
```

Comment:
no need this Mutex, see
https://github.com/lesismal/nbio/pull/43/files#r661936248